libraries/rush-lib/src/logic/base/BaseInstallManager.ts

```diff
@@ -390,20 +450,50 @@ export abstract class BaseInstallManager {
       this._rushConfiguration.commonRushConfigFolder,
       this._rushConfiguration.commonTempFolder
     );
+    if (this._rushConfiguration.hasSplitWorkspaceProject) {
+      const commonRushConfigSplitWorkspaceNpmrcPath: string = path.join(
+        this._rushConfiguration.commonRushConfigFolder,
+        '.npmrc-split-workspace'
+      );
+      if (!FileSystem.exists(commonRushConfigSplitWorkspaceNpmrcPath)) {
+        const splitWorkspaceNpmrcTemplatePath: string = path.join(
+          __dirname,
+          '../../../assets/split-workspace/.npmrc-split-workspace'
+        );
+        FileSystem.copyFile({
+          sourcePath: splitWorkspaceNpmrcTemplatePath,
+          destinationPath: commonRushConfigSplitWorkspaceNpmrcPath
+        });
+      }
+      process.env['_RUSH_COMMON_TEMP_SPLIT_FOLDER'] = this._rushConfiguration.commonTempSplitFolder;
```

Comment:
Add a code comment explaining what `_RUSH_COMMON_TEMP_SPLIT_FOLDER` is used for; it is not obvious